library/src/io/adc.c

```diff
@@ -24,6 +25,7 @@
 #define V_DIV_RATIO	11.0
 #define BATT_DEADZONE	1.0
 
+#define ADC2AI(x) (x==0?0:(x==1?1:(x==2?3:(x==3?2:(x==4?7:(x==5?6:(x==6?4:(x==7?5:-1))))))))
```

Comment:
This is an ugly hack. Not the desired approach. I need to figure out how to do a better job remapping the channels in a more portable way.

Reply:
Can we add a series of symlink entries into the device tree to create something like /dev/bone/adc/Px_y? I'd think that would be a better way to handle GPIOs than with the LED drivers like Deepak Khatri did for now.

Reply:
> This is an ugly hack. Not the desired approach. I need to figure out how to do a better job remapping the channels in a more portable way.

can we make ADC2AI(x) an inline function